docs/pages/access-controls/idps/saml-reference.mdx

```diff
@@ -140,7 +140,8 @@ of their role level permissions.
 
 ## Notes
 
-The SAML identity provider does not currently support [HSM](../../choose-an-edition/teleport-enterprise/hsm.mdx).
+SAML identity provider supports [HSM](../../choose-an-edition/teleport-enterprise/hsm.mdx)
```

Comment:
```suggestion
The SAML identity provider supports [HSM](../../choose-an-edition/teleport-enterprise/hsm.mdx).
```

Reply:
Are there any extra configuration steps a user needs carry out to integrate the SAML identity provider with HSM, or is it ready to go once they follow the steps in the HSM guide?